src/StyledApp.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
+body {
+  background-color: rgba(40, 42,53, 1);
+  
+  font-family: "Josefin Sans", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 200;
+  font-style: normal;
+}
+`;
+
+export const StyledImage = styled.img`
```

Comment:
change request: this file should contain only global styles, please move StyledImage styles where it's being used. If you are using these styles in couple of place please move it to project components scope.